data/js/values/values.js

```diff
@@ -9,10 +9,10 @@
     var nextToken;
 
     // if we're not on the final sidebar
-    if (parseInt(button.dataset.step) < 5) {
+    if (parseInt(button.dataset.step, 10) < 5) {
         // grab our next button and next token values
         nextButton = document.getElementById('next-button');
-        nextToken = document.querySelector('#token' + (parseInt(button.dataset.step) + 1));
+        nextToken = document.querySelector('#token' + (parseInt(button.dataset.step, 10) + 1));
```

Comment:
Could you use `getElementById` here? Could also use a template literal, if you wanted to use shiny things.